.gdbinit

```diff
@@ -270,6 +272,8 @@ def fetch_breakpoints(regular_only):
         breakpoint['condition'] = gdb_breakpoint.condition
         breakpoint['temporary'] = gdb_breakpoint.temporary
         breakpoint['hit_count'] = gdb_breakpoint.hit_count
+        breakpoint['pending'] = gdb_breakpoint.pending
+        breakpoint['visible'] = gdb_breakpoint.visible
```

Comment:
Do you need the visibility status for your own purpose? Besides, can this ever be `False`? The documentation is a bit obscure IMHO:

> This attribute tells whether the breakpoint is visible to the user when set, or when the ‘info breakpoints’ command is run. This attribute is not writable.

Reply:
Oops! That was an artifact that I meant to get rid of. (I put it in while working on the PR, but meant to take it out.) Will fix.